### DISCUSSION TITLE
Кнопки управления LaserGRBL

### DISCUSSION THREAD

**Discussion by toll101** [arkypita/LaserGRBL#1365] [Q&A]
Скажите, почему в программе LaserGRBL 4....  Нет переключателя количества миллиметров при одном нажатии?
Или как установить?
![Безимени-1](https://user-images.githubusercontent.com/49711876/110962606-d394bb80-8359-11eb-8912-15f7e2fe1b8b.jpg)
?

**Comment by arkypita** [comment]
Because YOU have activated "continuos jog".

**Reply by toll101** [reply]
Спасибо Большое.
Удачи в вашей работе!
